oak-run/src/main/java/org/apache/jackrabbit/oak/run/RevisionsCommand.java

```diff
@@ -535,8 +544,8 @@ private void collectDocument(RevisionsOptions options, Closer closer, String pat
         }
         gc.collectGarbageOnDocument(documentNodeStore, workingDocument, options.isVerbose());
 
-        //TODO: Probably we should output some details of fullGCStats. Could be done after OAK-10378
-        //gc.getFullGCStats();
+        System.out.println("Full GC Stats:");
+        System.out.println(gc.getFullGCStatsReport());
```

Comment:
just curious : above the report is indented with 4 spaces but not here - is this on purpose?

Reply:
Not on purpose, I will add them here too.